application/actions/gql.go

```diff
@@ -19,6 +19,8 @@ func gqlHandler(c buffalo.Context) error {
 
 	h := handler.GraphQL(gqlgen.NewExecutableSchema(gqlgen.Config{Resolvers: &gqlgen.Resolver{}}))
 	newCtx := context.WithValue(c.Request().Context(), domain.BuffaloContext, c)
+	newCtx = dataloader.GetDataLoaderContext(newCtx)
```

Comment:
Does this double-nesting mess up functions like `CurrentUser()` and `GetBuffaloContext()`?

Reply:
It doesn't seem to -- all the tests pass.  I assume we have tests for those.  I can look into it a bit.

Reply:
Good, no problem. I don't know if there's a test that would cover this, but the tests for those helper functions could be augmented to include a context with an extra layer.